packages/editor-ui/src/constants.workflows.ts

```diff
@@ -175,6 +175,7 @@ export const SAMPLE_SUBWORKFLOW_WORKFLOW: WorkflowDataWithTemplateId = {
 	nodes: [
 		{
 			id: 'c055762a-8fe7-4141-a639-df2372f30060',
+			typeVersion: 1.1,
```

Comment:
just for my understanding, do we need to bump the version here or do we load the latest version of the node?

Reply:
If you create a node from the sidebar, then we apply the latest version.
This is a template for sub-workflows that are created from parent workflow Resource Locator so we have to update it here since they are created by importing this JSON instead of adding nodes one by one programmatically.